c/disp_to_h.c

```diff
@@ -132,7 +132,7 @@ void disp_to_lonlatalt(double *lonlatalt, float *err,  // outputs
 }
 
 
-float squared_distance_between_3d_points(float a[3], float b[3])
+float squared_distance_between_3d_points(double a[3], double b[3])
```

Comment:
return should be double as well as x,y, and z

Reply:
I think that float should be enough here because the distances between 3D points of the same point cloud shouldn't be that big. But I don't mind changing to double, it doesn't hurt